### PR TITLE
added grids

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
127 0 0 1_5500_Documentation_grids_grids html

### CODE REVIEW

File: Documentation/grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+          </div>
+        </div>
+        <div>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253ETwo%2520column%2520grid%253C%252Fh2%253E%250A%253Cdiv%2520class%253D%2522grid-2-2%2522%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            style="
```
Comment (aatishpandey) [inline Documentation/grids/grids.html:97]: you can add styles in a stylesheet.

File: Documentation/grids/grids.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+              width: 400px;
```
Comment (aatishpandey) [inline Documentation/grids/grids.html:98]: can use rem here instead of px

File: Documentation/grids/grids.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <div class="grid-item"></div>
+          </div>
+        </div>
+        <div>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253EThree%2520column%2520grid%253C%252Fh2%253E%250A%253Cdiv%2520class%253D%2522grid-3-3%2522%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            style="
+              width: 400px;
+              height: 336px;
+              border: 0;
```
Comment (aatishpandey) [inline Documentation/grids/grids.html:126]: same here

**Review by aatishpandey — APPROVED** [review]
UI looks clean.
Added few suggestions.
Rest all is good.

**Review by sadathshariff — APPROVED** [review]
The Code looks clean and good

File: Documentation/grids/grids.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        </div>
+        <div>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253ETwo%2520column%2520grid%253C%252Fh2%253E%250A%253Cdiv%2520class%253D%2522grid-2-2%2522%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            style="
+              width: 400px;
+              height: 299px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
```
Comment (srejitk) [inline Documentation/grids/grids.html:102]: Refrain from keeping styles in the html page. Keep them seperate instead.

File: Stylesheets/components/grids.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    background-color: var(--gray-color);
+    margin:0.5rem;
+    height:5rem;
+    width:10rem;
+    border-radius:8px
+}
+
+.grid-3-3{
+    display:grid;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (srejitk) [inline Stylesheets/components/grids.css:18]: You can use repeat property here instead of doing it manually

**Review by ZuberDunge — APPROVED** [review]
Well-organized and structured code looks perfect, good to go from my side.

**Review by srejitk — APPROVED** [review]
Code looks good. Would suggest moving the CSS styles to a seperate CSS file.

File: Documentation/grids/grids.html (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply Documentation/grids/grids.html:97]: okay

File: Documentation/grids/grids.html (hunk 4 above)
Comment (Priyanshi-Tiwari) [reply Documentation/grids/grids.html:102]: okay

**Pull request merged**